src/library_glfw.js

```diff
@@ -979,14 +983,18 @@ var LibraryGLFW = {
       if (!win) return;
 
       if (GLFW.active.id == win.id) {
-        if (width == screen.width && height == screen.height) {
-          Browser.requestFullscreen();
-        } else {
-          Browser.exitFullscreen();
-          Browser.setCanvasSize(width, height);
-          win.width = width;
-          win.height = height;
-        }
+        // TODO HIGH YP: Browser.requestFullscreen does not work and raises an error:
+        //               Failed to execute 'requestFullscreen' on 'Element': API can only be initiated by a user gesture
+        //               Commented out and simply setting the canvas size seem to do the right thing
+        // if (width == screen.width && height == screen.height) {
+        //   Browser.requestFullscreen();
+        // } else {
+        //   Browser.exitFullscreen();
+        //   Browser.setCanvasSize(width, height);
+        //   // Implementation note: callback onCanvasResize will update values accordingly
+        // }
+        Browser.setCanvasSize(width, height);
+        // Implementation note: callback onCanvasResize will update values accordingly
```

Comment:
This seems like a fix for a separate issue.  Would you make making a separate PR for this?  I wonder if this one might be more testable?

Reply:
Sounds great. I will revert this changes from this PR. I created a separate [issue](https://github.com/emscripten-core/emscripten/issues/20600) for it and will create a separate PR for it.